x-pack/plugins/triggers_actions_ui/public/application/alert_table_config_registry.ts

```diff
@@ -28,7 +28,7 @@ export class AlertTableConfigRegistry {
   /**
    * Registers an object type to the type registry
    */
-  public register(objectType: AlertsTableConfigurationRegistry) {
+  public register<T extends object>(objectType: AlertsTableConfigurationRegistry) {
```

Comment:
I think we can remove it
```suggestion
  public register(objectType: AlertsTableConfigurationRegistry) {
```